src/IdentityManager/WebApi/Startup.cs

```diff
@@ -27,18 +27,11 @@ public Startup(IConfiguration configuration, IWebHostEnvironment environment)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            if(Environment.IsDevelopment())
-            {
-                
-            }
             services.AddCors(options =>
             {
                 options.AddDefaultPolicy(policy =>
                 {
-                    policy.AllowAnyHeader().AllowAnyMethod().AllowCredentials().WithOrigins(new[]
-                    {
-                        "http://localhost:3000"

```

Comment:
Any origin is not supported by browsers

Reply:
According to [this](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin) it should be supported. 

It is not allowed only with AllowCredentials. But we are not using cookies for authentication, so it should be fine.

Reply:
Ok